src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Templates/Microsoft.DotNet.UpgradeAssistant.Steps.Templates.csproj

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Remove="Templates\**\*.cs" />
+    <Compile Remove="Templates\**\*.vb" />
```

Comment:
Do you need `Compile` or `None`? I'm not sure how that works but compile might not be right since this is a csproj.

Reply:
Since we're using `Remove` I think both work. Switched to `None` so that the question is not as confusing.